diff -u a/systemd/nfs-config.service systemd/nfs-config.service
--- a/systemd/nfs-config.service	2016-12-17 11:43:47.062229400 +0100
+++ systemd/nfs-config.service	2016-12-17 11:44:14.737213168 +0100
@@ -10,4 +10,4 @@
 # incorporated.  Having "RemainAfterExit=no" (the default)
 # ensures this happens.
 RemainAfterExit=no
-ExecStart=/usr/libexec/nfs-utils/nfs-utils_env.sh
+ExecStart=/usr/lib/systemd/scripts/nfs-utils_env.sh
