Subject: Remove extra boggus whitespaces from .service files
From: Laurent Bigonville <bigon@debian.org>

--- a/systemd/nfs-server.service
+++ b/systemd/nfs-server.service
@@ -1,15 +1,15 @@
 [Unit]
 Description=NFS server and services
 DefaultDependencies=no
-Requires= network.target proc-fs-nfsd.mount
-Requires= nfs-mountd.service
+Requires=network.target proc-fs-nfsd.mount
+Requires=nfs-mountd.service
 Wants=rpcbind.socket
 Wants=nfs-idmapd.service
 
-After= local-fs.target
-After= network.target proc-fs-nfsd.mount rpcbind.socket nfs-mountd.service
-After= nfs-idmapd.service rpc-statd.service
-Before= rpc-statd-notify.service
+After=local-fs.target
+After=network.target proc-fs-nfsd.mount rpcbind.socket nfs-mountd.service
+After=nfs-idmapd.service rpc-statd.service
+Before=rpc-statd-notify.service
 
 # GSS services dependencies and ordering
 Wants=auth-rpcgss-module.service
