Index: nfs-utils-1.3.4/support/nsm/rpc.c
===================================================================
--- nfs-utils-1.3.4.orig/support/nsm/rpc.c
+++ nfs-utils-1.3.4/support/nsm/rpc.c
@@ -63,6 +63,10 @@
 #include "nsm.h"
 #include "sm_inter.h"
 
+#ifndef UINT16_MAX
+#   define UINT16_MAX      ((uint16_t)(65535U))
+#endif
+
 /*
  * Returns a fresh XID appropriate for RPC over UDP -- never zero.
  */
